vm/systemSmartContracts/governance.go

```diff
@@ -201,28 +197,27 @@ func (g *governanceContract) initV2(args *vmcommon.ContractCallInput) vmcommon.R
 	}
 
 	g.eei.SetStorage([]byte(ownerKey), args.CallerAddr)
-	g.ownerAddress = make([]byte, 0, len(args.CallerAddr))
-	g.ownerAddress = append(g.ownerAddress, args.CallerAddr...)
 
 	return vmcommon.Ok
 }
 
 // changeConfig allows the owner to change the configuration for requesting proposals
 //  args.Arguments[0] - proposalFee - as string
-//  args.Arguments[1] - minQuorum - 0-10000 - represents percentage
-//  args.Arguments[2] - minVeto   - 0-10000 - represents percentage
-//  args.Arguments[3] - minPass   - 0-10000 - represents percentage
+//  args.Arguments[1] - lostProposalFee - as string
+//  args.Arguments[2] - minQuorum - 0-10000 - represents percentage
+//  args.Arguments[3] - minVeto   - 0-10000 - represents percentage
+//  args.Arguments[4] - minPass   - 0-10000 - represents percentage
 func (g *governanceContract) changeConfig(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
-	if !bytes.Equal(g.changeConfigAddress, args.CallerAddr) {
+	if !bytes.Equal(g.ownerAddress, args.CallerAddr) {
 		g.eei.AddReturnMessage("changeConfig can be called only by owner")
 		return vmcommon.UserError
 	}
 	if args.CallValue.Cmp(zero) != 0 {
 		g.eei.AddReturnMessage("changeConfig can be called only without callValue")
 		return vmcommon.UserError
 	}
-	if len(args.Arguments) != 4 {
-		g.eei.AddReturnMessage("changeConfig needs 4 arguments")
+	if len(args.Arguments) != 5 {
```

Comment:
[comment]

Would've been cleaner to use consts for these values, although they are not used in this file in other places.